docs/installation.md

```diff
@@ -4,6 +4,38 @@ Installing the `kubectl-kjob` plugin, `kjobctl`.
 
 ## Installing 
 
+### From Release Binaries
+
+#### 1. Download the latest release:
+
+On Linux:
+```bash
+# AMD64 / x86_64
+curl -Lo ./kubectl-kjob https://github.com/kubernetes-sigs/kjob/releases/download/v0.1.0/kubectl-kjob-linux-amd64
```

Comment:
Please use separate boxes per architecture so that the copy button works as expected:
![image](https://github.com/user-attachments/assets/cdadad43-0fdd-4156-9fbf-80017fdd67c3)

Reply:
Done